DOCS/man/options.rst

```diff
@@ -109,7 +109,7 @@ Playback Control
 
     ``#c`` seeks to chapter number c. (Chapters start from 1.)
 
-    ``none`` resets any previously set option (useful for libmpv).
+    ``no`` or ``none`` resets any previously set option (useful for libmpv).
```

Comment:
Maybe document `none` as deprecated alias for `no`, so we can remove it in the future.